orttraining/orttraining/training_ops/cpu/triton/triton_op.cc

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Not sure it's the right place to put triton-related files under CPU folder.

Reply:
The impl uses OpKernelContextInternal, which is not visible from CUDA lib.

Reply:
Why do we need OpKernelContextInternal? Unless I am missing something, everything that is needed here is available via OpKernelContext

Reply:
GetInputMLValue and SetOutputMLValue.